.github/workflows/test_security.yml

```diff
@@ -0,0 +1,96 @@
+name: Security test workflow for k-NN
```

Comment:
Rename this to, test k-NN on Secure Cluster

Reply:
ack